scripts/zones/Metalworks/npcs/Iron_Eater.lua

```diff
@@ -14,6 +14,48 @@ local ID = require("scripts/zones/Metalworks/IDs")
 
 local wsQuest = tpz.wsquest.steel_cyclone
 
+local TrustMemory = function(player)
+    local memories = 126
```

Comment:
Maybe set this to 0 until the memories are implemented so people don't get confused~